### PR TITLE
Add combine_all_yearly_contributions query parameter to fetch and combine all yearly contributions, not only recent contributions

### LINKED ISSUE #6: List all contribution, not only recent contribution
Currently this card use `repositoriesContributedTo` to fetch contributions. However, as discussed at https://github.com/orgs/community/discussions/24350,  `repositoriesContributedTo` only lists recent contribution.

In order to list all contribution, we can instead use `contributionsCollection`.

### PR DESCRIPTION
**Pull request by Atry**
Rendering result on my local server:

<details>
  <summary>http://localhost:9999/api?username=Atry</summary>
  <p>
    
  </p>
</details>
<details>
  <summary>http://localhost:9999/api?username=Atry&combine_all_yearly_contributions=true</summary>
  <p>
    
  </p>
</details>

This PR fixes #6

### CODE REVIEW

**Comment by HwangTaehyun** [comment]
@Atry Thank you for PR! I will look into it!

File: README.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ To hide specific ranks, you can pass a query parameter `&hide=` with comma-separ
 ![Taehyun's GitHub Repository Contribution stats](https://github-contributor-stats.vercel.app/api?username=HwangTaehyun&hide=B,B%2B)
 ```
 
+### Including all contributions, not only recent contributions
+By default, the dard is generated from GitHub's GraphQL API `repositoriesContributedTo`, which only includes recent contributions. To include all contributions, add `&combine_all_yearly_contributions=true` query parameter, which will let the card be generated from GitHub's GraphQL API `contributionsCollection`, including all contributions.
```
Comment (HwangTaehyun) [inline README.md:39]: Could you fix ```dard``` typo?

**Comment by HwangTaehyun** [comment]
@Atry Thank you for contribution!!

**Comment by Atry** [comment]
@HwangTaehyun Would you mind also reviewing other pull requests?

**Pull request merged**